packages/formstr-app/src/containers/CreateForm/components/Validation/Number.tsx

```diff
@@ -0,0 +1,47 @@
+import { AnswerTypes } from "@formstr/sdk/dist/interfaces";
+import NumberInput from "./NumberInput";
+import { IProps } from "./typeDefs";
+
+function Number({ answerSettings, handleAnswerSettings }: IProps) {
+  const numberConstraints =
+    answerSettings.validationRules?.[AnswerTypes.number]?.[0] as { min: number; max: number };
+
+  return (
+    <div>
+      <NumberInput
+        label="Min:"
+        value={numberConstraints?.min}
+        onChange={(val) =>
+          handleAnswerSettings({
+            validationRules: {
```

Comment:
should be an array of type `{label: string; errorMessage: string; [key: string]: uknown }`

Reply:
added new interface